extensions/guacamole-auth-ldap/src/main/java/org/apache/guacamole/auth/ldap/ConfigurationService.java

```diff
@@ -344,4 +344,20 @@ public int getOperationTimeout() throws GuacamoleException {
         );
     }
 
+    /**
+     * Returns names for custom LDAP user attributes.
+     *
+     * @return
+     *     LDAP user attributes as defined in the guacamole.properties file
+     *     as ldap-user-attributes: ''
```

Comment:
This comment just strikes me as a little odd - I would drop the example of its guacamole.properties definition and just allow that to be documented in the manual and make this more like the other `@return` tags for other properties.